presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -1046,6 +1047,14 @@ public Void visitAssignUniqueId(AssignUniqueId node, Integer indent)
         }
 
         @Override
+        public Void visitGroupReference(GroupReference node, Integer indent)
```

Comment:
Is this just for debugging? GroupReference nodes should not exists outside of IterativeOptimizer. Therefore, they should never make it to an EXPLAIN call.